src/components/Header/styles.js

```diff
@@ -0,0 +1,75 @@
+import styled from "styled-components";
+
+export const HeaderContainer = styled.nav`
+  display: flex;
+  align-items: center;
+  height: 8rem;
+  justify-content: space-between;
+
+  @media screen and (max-width: 1024px) {
+    justify-content: center;
+    align-items: center;
+  }
+
+  @media screen and (max-width: 600px) {
+    height: 6rem;
+  }
+`;
+
+export const Links = styled.ul`
+  display: flex;
+  text-transform: uppercase;
+  cursor: pointer;
+`;
+
+export const ItemLink = styled.li`
+  display: flex;
+  align-items: center;
+  font-weight: bold;
+  gap: 2.5rem;
+
+  @media screen and (min-width: 1024px) {
+    gap: 5rem;
+  }
+
+  @media screen and (max-width: 600px) {
+    gap: 0.8rem;
+  }
+`;
+
+export const Link = styled.a`
+  font-family: "Eater", cursive;
+  letter-spacing: 0.145rem;
+  text-decoration: none;
+  font-size: 1.125rem;
```

Comment:
Acho que 1.125rem ficou muito grande, achei que o tamanho 1rem ficou bem legal ja :3
```suggestion
  transition: 0.3s
```